lib/net/nvnetdrv/nvnetdrv.c

```diff
@@ -441,17 +447,25 @@ int nvnetdrv_init (size_t rx_buffer_count, nvnetdrv_rx_callback_t rx_callback)
     }
 
     // Allocate memory for TX and RX ring descriptors.
-    void *descriptors = MmAllocateContiguousMemoryEx((RX_RING_SIZE + TX_RING_SIZE) * sizeof(struct descriptor_t), 0,
+    void *descriptors = MmAllocateContiguousMemoryEx((g_rxRingSize + g_txRingSize) * sizeof(struct descriptor_t), 0,

```

Comment:
For performance reasons, you may want to align this pointer on a 64-byte boundary.

Reply:
`MmAllocateContiguousMemoryEx` should allocate physical memory pages, so I'm not sure if this could ever *not* be aligned to 0x1000 (which is a multiple of 0x40)?

Reply:
Yeah it's always page-aligned. `MmAllocateContiguousMemoryEx` has an alignment parameter, but that only does something if you want a POT alignment bigger than 4KiB.